packages/suite-desktop-core/src/config.ts

```diff
@@ -25,6 +25,11 @@ export const allowedDomains = [
     'trezor-cardano-mainnet.blockfrost.io',
     'trezor-cardano-preview.blockfrost.io',
     'blockfrost.dev',
+    'eth-goerli.public.blastapi.io',
+    'ethereum-holesky.publicnode.com',
+    'mainnet.infura.io',
```

Comment:
still needed?

Reply:
Blockbook data is not used for staking/unstaking/claiming yet. AFAIR, we've decided with you to merge everything as is. And then keep going with the rest of changes. So, this is an upcoming change.